avm/res/cognitive-services/account/main.bicep

```diff
@@ -135,6 +136,30 @@ param secretsExportConfiguration secretsExportConfigurationType?
 @description('Optional. Enable/Disable project management feature for AI Foundry.')
 param allowProjectManagement bool?
 
+@description('Optional. Enable commitment plan for disconnected container for Neural TTS')
+param isCommitmentPlanForDisconnectedContainerEnabledForNeuralTTS bool = false
+
+@description('Optional. Commitment plan properties for disconnected container for Neural TTS')
+param commitmentPlanForDisconnectedContainerForNeuralTTS disconnectedContainerCommitmentPlanType?
+
+@description('Optional. Enable commitment plan for disconnected container for STT')
+param isCommitmentPlanForDisconnectedContainerEnabledForSTT bool = false
+
+@description('Optional. Commitment plan properties for disconnected container for STT')
+param commitmentPlanForDisconnectedContainerForSTT disconnectedContainerCommitmentPlanType?
+
+@description('Optional. Enable commitment plan for disconnected container for Custom STT')
+param isCommitmentPlanForDisconnectedContainerEnabledForCustomSTT bool = false
+
+@description('Optional. Commitment plan properties for disconnected container for Custom STT')
+param commitmentPlanForDisconnectedContainerForCustomSTT disconnectedContainerCommitmentPlanType?
+
+@description('Optional. Enable commitment plan for disconnected container for Add-On')
+param isCommitmentPlanForDisconnectedContainerEnabledForAddOn bool = false
+
+@description('Optional. Commitment plan properties for disconnected container for Add-On')
+param commitmentPlanForDisconnectedContainerForAddOn disconnectedContainerCommitmentPlanType?
+
```

Comment:
These names are super confusing. We need a cleaner structure here Can you try using an object these related params?